packages/@aws-cdk/aws-stepfunctions-tasks/lib/sagemaker-create-endpoint-config-task.ts

```diff
@@ -0,0 +1,152 @@
+import * as iam from '@aws-cdk/aws-iam';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import { Stack } from '@aws-cdk/core';
+import { getResourceArn } from './resource-arn-suffix';
+import { DataCaptureConfig, ProductionVariants } from './sagemaker-task-base-types';
+
+/**
+ *  @experimental
```

Comment:
Add the following text and to all other modules to which this apply.

```suggestion
 *  @experimental - since the API may change when higher level constructs in the `aws-sagemaker` module is available
```